snap/info.go

```diff
@@ -167,7 +167,12 @@ type Info struct {
 	Publisher   string
 
 	Screenshots []ScreenshotInfo
-	Channels    map[string]*ChannelSnapInfo
+
+	// The flattended channel map with $track/$risk
+	Channels map[string]*ChannelSnapInfo
+
+	// The ordered list of tracks that contains channels
```

Comment:
Nitpick: s/contains/contain/ ("tracks that contain channels", the list itself doesn't contain channels)